src/ast.rs

```diff
@@ -46,9 +46,14 @@ impl<'a> Ast<'a> {
         break;
       }
     }
-
     Ok(Ast { statements })
   }
+
+  pub fn print(&self) {
```

Comment:
What are these print functions for? What about the stopstd approach wasn't working?